logql/v2/expr.y

```diff
@@ -315,7 +333,7 @@ binaryOpOptions:
 selector:
                 OPEN_BRACE matchers CLOSE_BRACE { $$ = $2 }
         |       OPEN_BRACE matchers error       { $$ = $2 }
-        |       OPEN_BRACE error CLOSE_BRACE    {         }
+        |       OPEN_BRACE CLOSE_BRACE          {         }
```

Comment:
Is this an accidentally changed?

Reply:
Nope. The error handling changed in `logql` such that an empty query should no longer cause a syntax error.